templates/.phpcs.xml.dist

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards based custom ruleset for your plugin">
+    <description>Generally-applicable sniffs for WordPress plugins.</description>
+
+    <!-- What to scan -->
+    <file>.</file>
+    <exclude-pattern>vendor/</exclude-pattern>
```

Comment:
Be aware that `exclude-pattern` values are interpreted as regular expressions, so this snippet would exclude *every* directory which included the string `vendor/`, i.e. `/some-other-vendor/` would also be excluded.

While this may be something which will rarely cause issues, you can stabilize this to just the Composer `vendor` directory by using something along the lines of `^/vendor/*` or the slightly more generic `./vendor/`

Reply:
For more information about how the `exclude-pattern`s are interpreted, see: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-files-and-folders

Reply:
Darn... my bad. The more generic version should be `/vendor/`. The `.` which I'd inadvertently placed in front of it will not be interpreted as root, but as a regex wildcard, so, while it shouldn't give any problems, it also isn't necessary.